console/console-init/ui/src/modules/connection-detail/components/MessagesDetailHeaderAttributes/MessagesDetailHeaderAttributes.tsx

```diff
@@ -29,9 +29,9 @@ export const MessagesDetailHeaderAttributes: React.FunctionComponent<IMessagesDe
   isMobileView
 }) => {
   return (
-    <Split id="message-detail">
+    <Split id="message-detail-split">
```

Comment:
Again, split and splititem makes this id too technical and PF specific.

Reply:
split and split item don't really need an id. Let me know if I should still change it